packages/runtime/datastore/src/localChannelContext.ts

```diff
@@ -332,3 +331,21 @@ export class LocalChannelContext extends LocalChannelContextBase {
 		};
 	}
 }
+
+/**
+ * Deep clones a snapshot tree.
+ * 
+ * TODO: Investigate replacing this with a deep clone utility.
+ * This is a temporary solution to avoid issues with lodash deepClone and ungap structuredClone.
```

Comment:
Might be nice to mention what the issue is, for the next reader. "ISnapshotTrees can have a `blobContents` property on them, which holds ArrayBuffers, and those cannot be cloned by the ungap polyfill. Using lodash incurred a significant bundle size regression."